optaplanner-wb-webapp/.gitignore

```diff
@@ -1,10 +1,9 @@
 /target
 /local
 /repositories
-/src/main/gwt-unitCache/
-/src/main/webapp/WEB-INF/classes/
-/src/main/webapp/WEB-INF/lib/
-/src/main/webapp/org.optaplanner.workbench.OptaPlannerWorkbench/
+
+#https://intellij-support.jetbrains.com/hc/en-us/community/posts/206975205-Where-does-the-overlays-directory-come-from-
+/overlays
```

Comment:
Anyway we can configure the pom.xml in such a way that it's located in /target/overlays instead??
So "sync from maven pom" in intellij should just have the correct path (/target/overlays), without any manual changes.

Reply:
As I understand it, the `overlays` dir is created by IDEA, not Maven. So it can't be configured in POM. I don't know if the location can be changed in IDEA.